.gitpod/readme.md

```diff
@@ -0,0 +1,49 @@
+# Welcome to Starport ✨
+
+Starport is an easy to use CLI tool for creating sovereign blockchains. Blockchains created with Starport use Cosmos SDK, the world's most widely-used blockchain application framework.
+
+In front of you is a browser-based development environment. At the bottom of the screen you can see a terminal window, it has `starport` binary pre-installed.
+
+## Quick start
+
+Create a new blockchain and start a node in development:
+
+```
+starport app github.com/user/hello
+
+cd hello
+
+starport serve
+```
+
+## Next steps
+
+📺 **[Introduction](https://www.youtube.com/watch?v=5RqAIE0b8Kw)**: watch a quick introduction to Starport.
+
+🧑‍🏫 **[Tutorials](https://tutorials.cosmos.network)**: learn by building a simple IBC-enabled module, nameservice or a decentralized exchange (DEX).
+
+📕 **[Starport Documentation](https://docs.starport.network)**: explore features of Starport.
+
+📚 [Cosmos SDK Documentation](https://docs.cosmos.network): learn more about the framework.
+
+⭐️ [Starport on Github](https://github.com/tendermint/starport): submit an issue or contribute to the source code.
+
+## Starport features
+
+* Scaffold modules, messages, types with CRUD operations, IBC packets and more
```

Comment:
```suggestion
* Scaffold modules, messages, types with CRUD operations, IBC packets, and more
```

Reply:
hi Denis, you know I'm passionate about the serial comma (aka the Harvard comma)